Comp2023/src/main/java/frc/robot/commands/AutoPreloadHigh.java

```diff
@@ -26,21 +25,15 @@ public AutoPreloadHigh(Elbow elbow, Extension extension, Wrist wrist, Gripper gr
 
         // @formatter:off
         new PrintCommand(getName() + ": AUTO: Move Arm for Preload"),        
-        new ParallelDeadlineGroup(
-            new ArmSetHeightScoreHigh(elbow, extension, wrist)
-        ),
+        new ArmSetHeightScoreHigh(elbow, extension, wrist),

```

Comment:
I thought we wanted to change this so that the wrist was set at stow and then lowered to 90 degrees before we ran gripper?

Reply:
I think this is a product of your branch (with the fix) being merged into main, but these PRs are from Alex (and my) branch. I'll try to do a merge to update the PR.

Reply:
I've merged from main back to this branch. I only lightly reviewed the merge conflicts. Please re-review this file in detail to see if it is correct now. If not, please checkout and edit the branch directly.